genbadge/utils_flake8.py

```diff
@@ -115,7 +117,14 @@ def parse_flake8_stats(stats_txt  # type: str
     for line in stats_txt.splitlines():
         match = RE_TO_MATCH.match(line)
         if not match:
-            warn("Line in Flake8 statistics report does not match template and will be ignored: %r" % line)
+            smatch = SOURCE_MATCH.match(line)
```

Comment:
```suggestion
            # Other supported lines ?
            smatch = SOURCE_MATCH.match(line)
```

Reply:
Yes, thank you @smarie, I did not realize that it wasn't being caught because I did not audit and verify the counts.

I have quite a few warnings from several flake8 plugins.  I can organize the ones you don't already have into tests/reports/flake8/flake8stats.txt and cross-check the counts.  I will report back on my findings, and if I introduce any changes to the code I will provide tests.

It seems to me that I should also provide tests for the ":" cases, so I will try testing against that also.  I will try to turn this around over the next few days.

Reply:
That would be great @rnickle ! My guess is that with one or two additional test files it should cover most of the variants, but I let you find investigate. Let me know !